### PR TITLE
fix benchmarks

### PR DESCRIPTION
**Pull request by kinosang**
This PR fix benchmarks

**Type of change**

* [x] Bug Fix (non-breaking change which fixes an issue)
* [x] This change requires a benchmark

**Checklist**

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] New and existing unit tests pass locally with my changes
* [x] New and existing benchmarks pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream modules
* [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pallets/ad/src/benchmarking.rs [hunk 1]
```diff
@@ -79,185 +69,150 @@ benchmarks! {
...
-        assert_eq!(Tag::<T>::tags_of(&ad).len(), tags.len());
+        assert_eq!(Tag::<T>::tags_of(&ad).len(), 1);
     }
 
     add_budget {
         let caller: T::AccountId = whitelisted_caller();
 
-        let max = BalanceOf::<T>::max_value();
         let min = <T as parami_did::Config>::Currency::minimum_balance();
+        let pot = min.saturating_mul(1_000_000_000u32.into());
```
Comment (coderfengyun) [inline pallets/ad/src/benchmarking.rs:106]: 为什么要mul 1e9呢？

File: pallets/ad/src/benchmarking.rs [hunk 2]
```diff
@@ -79,185 +69,150 @@ benchmarks! {
...
 
     bid {
         let caller: T::AccountId = whitelisted_caller();
-
         let kol: T::AccountId = account("kol", 1, 1);
 
-        let max = BalanceOf::<T>::max_value();
         let min = <T as parami_did::Config>::Currency::minimum_balance();
-        let pot = min.saturating_mul(1_000_000u32.into());
+        let pot = min.saturating_mul(1_000_000_000u32.into());
```
Comment (coderfengyun) [inline pallets/ad/src/benchmarking.rs:137]: 同上

File: pallets/linker/src/benchmarking.rs [hunk 3]
```diff
@@ -71,7 +63,7 @@ benchmarks! {
 
         let max = BalanceOf::<T>::max_value();
         let min = T::Currency::minimum_balance();
-        let pot = min.saturating_mul(1_000_000u32.into());
+        let pot = min.saturating_mul(100_000u32.into()).saturating_mul(1_000_000u32.into());
```
Comment (coderfengyun) [inline pallets/linker/src/benchmarking.rs:66]: 这里的1e5和1e6代表什么含义呢？

File: pallets/ad/src/benchmarking.rs (hunk 1 above)
Comment (kinosang) [reply pallets/ad/src/benchmarking.rs:106]: 因为 runtime 里的要求增加了

File: pallets/ad/src/benchmarking.rs (hunk 2 above)
Comment (kinosang) [reply pallets/ad/src/benchmarking.rs:137]: 同上

File: pallets/linker/src/benchmarking.rs (hunk 3 above)
Comment (kinosang) [reply pallets/linker/src/benchmarking.rs:66]: 代表 runtime 里面配置的 deposit 要求

File: pallets/ad/src/benchmarking.rs (hunk 1 above)
Comment (coderfengyun) [reply pallets/ad/src/benchmarking.rs:106]: 有办法定义为常量吗？在runtime和benchmark两边共享?

File: pallets/linker/src/benchmarking.rs (hunk 3 above)
Comment (coderfengyun) [reply pallets/linker/src/benchmarking.rs:66]: 同样，能否定义常量runtime与benchmark都使用常量呢？

File: pallets/ad/src/benchmarking.rs (hunk 1 above)
Comment (kinosang) [reply pallets/ad/src/benchmarking.rs:106]: 可以放在 parami-primitives 里

Comment (coderfengyun) [reply pallets/ad/src/benchmarking.rs:106]: OK，挪一下？

**Pull request merged**